CRM/Civicase/Hook/BuildForm/ButtonsLableChange.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * Changes the label of buttons in the create pdf popup.
+ */
+class CRM_Civicase_Hook_BuildForm_ButtonsLableChange {
+
+  /**
+   * Updates form buttons labels.
+   *
+   * @param CRM_Core_Form $form
+   *   Form Class object.
+   * @param string $formName
+   *   Form Name.
+   */
+  public function run(CRM_Core_Form &$form, $formName) {
+    if (!$this->shouldRun($formName)) {
+      return;
+    }
+    $buttonGroup = $form->getElement('buttons');
+    $buttons = $buttonGroup->getElements();
```

Comment:
Can we have something like this:
```php
//An array of Button name and new label:
    $buttonLabels = ['_qf_PDF_submit_preview' => 'Download Preview', '_qf_PDF_upload' => 'Save and create document'];
    foreach ($buttons as $button) {
      $attr = $button->getAttributes();
      if (!empty($attr['name']) && isset($buttonLabels[$attr['name']])) {
        $button->setValue(ts($buttonLabels[$attr['name']]));
      }
    }
```

We can get rid of the extra else and duplicate logic that way and if we need to rename more buttons in the future, we just add one more element to the array.

Also Note that I added the translation function for the button labels.